besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -991,9 +986,7 @@ void setBesuConfiguration(final BesuConfiguration pluginCommonConfiguration) {
 
   private BesuCommand handleStandaloneCommand() {
     standaloneCommands = new StandaloneCommand();
```

Comment:
Is folding StandaloneCommand back into Besu Command part of this PR or a follow on PR?

Reply:
I missed that bit, I've added it into same PR.